java/runtime/src/main/java/io/ray/runtime/runner/worker/DefaultDriver.java

```diff
@@ -1,30 +0,0 @@
-package io.ray.runtime.runner.worker;
-
-import io.ray.api.Ray;
-
-/**
- * The main function of DefaultDriver.
- */
-public class DefaultDriver {
```

Comment:
Why delete `DefaultDriver`?

Reply:
It's not actually used. The the code is out-dated. I think we can re-add it when we introduce job submission.